### DISCUSSION TITLE
Update TOS to explicitly ban sysmeds and exclusionists

### DISCUSSION THREAD

**Discussion by hasrock36** [crabber-net/crabber#118] [Ideas]
Over the past few days Crabber has become a bit of a safe space for systems coming in from other platforms, but now we're being followed by the very people we left other platforms because of. To really set Crabber apart and to help safeguard the systems that have joined I suggest banning sysmeds outright

**Comment by RusanyaCollective** [comment]
Agreed. 👍

Being endogenic (a system formed by something other than trauma) is first and foremost a belief, even ignoring the fact psychology is only interested in that which it can pathologize.

You also can't tell us, an endogenic, that we don't exist when we are typing here as you read.

The idea that we are just "not remembering" trauma doesn't work when we already know all of our demons. We check and recheck several times in a month (no thanks to sysmedicalists and anti-endos in particular for this one) and have yet to find anything over the two or three years that we have known about our existence.

We do not owe you our life story to gain some pithy form of "validation" that can be revoked if we so much as step on the mat instead of the path of eggshells these anti-endos have laid bare.

Being anti-endogenic, by way of hating endogenic people for the act of existing, breathing, and talking about our own realities, are bigots — ableists in particular — and should be treated on the same level of other ableists: by banning them, and deplatforming them.

EDITED for minor typos.

**Comment by hqnna** [comment]
I'm fine with this change, poking @jakeledoux to get their opinion on it.

**Comment by ImSerei** [comment]
well doesnt "be respectful to users" and "no hateful ideology" in the TOS cover it already

**Comment by RusanyaCollective** [comment]
> well doesnt "be respectful to users" and "no hateful ideology" in the TOS cover it already

Yes, generally, but unfortunately most people aren't even aware this is an issue, so having it be explicitly in the ToS might be more helpful ioo.

Source: we're an endogenic who has been the target of Twitter harassment campaigns...for being g endogenic and defending our existence

//not mad

**Comment by jakeledoux** [comment]
I briefly addressed this on Discord earlier and you can see my message here:
<img width="541" alt="Screen Shot 2021-10-25 at 4 57 08 PM" src="https://user-images.githubusercontent.com/5758561/138769718-51ff57f3-2915-4be0-a05d-17a1849cd51d.png">

In short: Hate speech and harassment already is against the TOS. Please report any rule-breaking accounts to contact@crabber.net and I will do my best to take care of it.

**Comment by hqnna** [comment]
I would also like people to keep [this](https://crabber.net/user/hungryghosts/status/39461/) molt in mind when discussing this, not everyone has knowledge about sysmedicalism and plurality, etc, so adding specific things like this would be confusing to some.
